pkg/neutronapi/initcontainer.go

```diff
@@ -29,68 +30,85 @@ func GetInitContainer(init InitContainer) []corev1.Container {
 		securityContext.Privileged = &trueVar
 	}
 
-	return []corev1.Container{
+	envs := []corev1.EnvVar{
 		{
-			Name:            "init",
-			Image:           init.ContainerImage,
-			SecurityContext: securityContext,
-			Command: []string{
-				"/bin/bash", "-c", "/usr/local/bin/container-scripts/init.sh",
-			},
-			Env: []corev1.EnvVar{
-				//	{
-				//		Name: "TransportURL",
-				//		ValueFrom: &corev1.EnvVarSource{
-				//			SecretKeyRef: &corev1.SecretKeySelector{
-				//				LocalObjectReference: corev1.LocalObjectReference{
-				//					Name: init.NeutronSecret,
-				//				},
-				//				Key: "TransportUrl",
-				//			},
-				//		},
-				//	},
-				{
-					Name:  "DatabaseHost",
-					Value: init.DatabaseHost,
+			Name: "TransportURL",
+			ValueFrom: &corev1.EnvVarSource{
+				SecretKeyRef: &corev1.SecretKeySelector{
+					LocalObjectReference: corev1.LocalObjectReference{
+						Name: "rabbitmq-transport-url-neutron-neutron-transport",
+					},
+					Key: "transport_url",
 				},
-				{
-					Name:  "Database",
-					Value: init.Database,
+			},
```

Comment:
It looks like this is duplicated.  Lines 88-101 seem to handle whether `TransportURL` should be included in the env vars.